lib/error-handler.js

```diff
@@ -113,11 +114,7 @@ function fallbackErrorHandler (error, reply, cb) {
     // error is always FST_ERR_SCH_SERIALIZATION_BUILD because this is called from route/compileSchemasForSerialization
     reply.log.error({ err, statusCode: res.statusCode }, 'The serializer for the given status code failed')
     reply.code(500)
-    payload = serializeError({
-      error: statusCodes['500'],
-      message: err.message,
-      statusCode: 500
-    })
+    payload = serializeError(new FST_ERR_FAILED_ERROR_SERIALIZATION(err.message, error.message))
```

Comment:
I would treat this change as "sensible" because some users may not expect the additional error detail

I'm +1 to ship it anyway